spec/spec_helper.rb

```diff
@@ -7,38 +7,12 @@
 
 require 'rubygems'
 require 'bundler/setup'
-
-MINIMUM_COVERAGE = 71
-
-unless ENV['COVERAGE'] == 'off'
-  require 'simplecov'
-  require 'simplecov-rcov'
-  require 'coveralls'
-  Coveralls.wear!
-
-  SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter[
-    SimpleCov::Formatter::RcovFormatter,
-    Coveralls::SimpleCov::Formatter
-  ]
-  SimpleCov.start do
-    add_filter '/vendor/'
-    add_filter '/spec/'
-    add_group 'lib', 'lib'
-  end
-  SimpleCov.at_exit do
-    SimpleCov.result.format!
-    percent = SimpleCov.result.covered_percent
-    unless percent >= MINIMUM_COVERAGE
-      puts "Coverage must be above #{MINIMUM_COVERAGE}%. It is #{"%.2f" % percent}%"
-      Kernel.exit(1)
-    end
-  end
-end
+require 'support/coverage_loader'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.